### ISSUE TITLE
Create profile side nav not a11y friendly

### ISSUE THREAD

**Issue by dynamic11** [CDH-Studio/UpSkill#1494]
**Describe the bug**
The side bar in create profile is not screen reader friendly:
- it has empty divs that get picked up when tabbing
- icons get read outloud 

**To Reproduce**

1. Go to "create profile"
2. tab through side bar

**Expected behavior**
The screen reader should avoid undescriptive icons and unused divs should be removed

**Screenshots**
no screen shot necessary
